# An array of points

std_msgs/Int32 trasmittor_id
geometry_msgs/Point[] frontier_points
